src/ItemNote.ts

```diff
@@ -27,6 +27,9 @@ import { ensureFolderExists, getPocketItemPocketURL } from "./utils";
 
 const DEFAULT_ITEM_NOTES_FOLDER = "/";
 
+// Windows allows at-most 255 characters. We use a safely shorter length for additional compatibility.
```

Comment:
This is actually a more widespread limitation, i.e. most filesystems allow at most 255 characters. I'd change the comment to reflect that fact.

Also, please change the comment to say that "five chars are reserved for handling duplicate filenames".